sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+# tag_based_pnp_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/tag_based_pnp_calibrator.md).
+
+## Purpose
+
+The package `tag_based_pnp_calibrator` allows extrinsic calibration among the camera and lidar sensors used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+The `tag_based_pnp_calibrator` utilizes the PnP (Perspective-n-Point) algorithm, a computer vision method that finds the best match between a set of 3D points (from the lidar) and their corresponding 2D projections (in the camera images), to accurately calculate the transformation between the camera and lidar. To run this package, you also need to execute the `apriltag_ros` and the `lidartag` packages to calculate the transformation.
+
+The `apriltag_ros` package detects AprilTag markers from an image and outputs the detection results. Conversely, the `lidartag` package detects LidarTag markers and outputs its detection results.
```

Comment:
I see:
 - LiDARTag
 - LidarTag
 - Lidartags
 - lidartag

Please make it consistent
Although I am inclined to use `lidartag` (due to ros2 naming conventions), I will leave the choice to you.

Reply:
@vividf 
friendly reminder

Reply:
@knzo25 
Thanks! After I fixed them, I didn't see this comment.
https://github.com/tier4/CalibrationTools/pull/164/commits/8cc5d621ab26a27f6d02d53d64d754a50b2b3601